### ISSUE TITLE
error for version 1.2.4

### ISSUE THREAD

**Issue by lijing28101** [EI-CoreBioinformatics/mikado#268]
Hi, I'm running mikado 1.2.4 to combine gtf from cufflinks, stringtie and CLASS2.
Here is the error:
```
$ mikado prepare --json-conf configuration.yaml -p 16 -od mikado_out
/work/LAS/mash-lab/jing/bin/Anaconda3/envs/pyrpipe_human/lib/python3.6/site-packages/Mikado/configuration/configurator.py:529: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/lofor full details.
  scoring = yaml.load(scoring_file)
Process AnnotationParser-12:
Traceback (most recent call last):
  File "/work/LAS/mash-lab/jing/bin/Anaconda3/envs/pyrpipe_human/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/work/LAS/mash-lab/jing/bin/Anaconda3/envs/pyrpipe_human/lib/python3.6/site-packages/Mikado/preparation/annotation_parser.py", line 98, in run
    strand_specific=strand_specific)
  File "/work/LAS/mash-lab/jing/bin/Anaconda3/envs/pyrpipe_human/lib/python3.6/site-packages/Mikado/preparation/annotation_parser.py", line 448, in load_from_gtf
    exon_lines[row.id]["tid"] = row.transcript
KeyError: None
```
I didn't see this error when I combine other files from stringtie and cufflinks. 
Could you help me to figure out the problem? Or when will the database error (#264 ) can be fixed?

**Comment by swarbred** [comment]
@lijing28101 

have a look at your CLASS2 file
```
exon_lines[row.id]["tid"] = row.transcript
```
would seem to indicate it's not formated as mikado prepare expects. 

Perhaps run gffread on the file with the -T option and see if that output works

**Comment by lucventurini** [comment]
This should now be fixed in version 2.0rc2 on both [PyPI](https://pypi.org/project/Mikado/2.0rc2/) and [BioConda](https://anaconda.org/bioconda/mikado). Please repoen if the issue represents itself with the new version!
